src/__tests__/Utils.test.ts

```
@@ -88,6 +90,22 @@ export const createMockTransactionReceipt = (overrides = {}): TransactionReceipt
+    getVoteSubmittedEvents: jest.fn(() => Promise.resolve([])),
+    getTurnEndedEvents: jest.fn(),
+    getGameCreatedEvents: jest.fn(),
+    getPlayerJoinedEvents: jest.fn(),
+    getGameStartedEvents: jest.fn(),
+    getGameOverEvents: jest.fn(),
+    getMAOInstances: jest.fn(),
+    queryInstances: jest.fn()
+  };
+
+  return mockClient as unknown as EnvioGraphQLClient;
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)